packages/app-desktop/gui/ConfigScreen/ConfigScreen.tsx

```diff
@@ -20,11 +20,20 @@ import ToggleAdvancedSettingsButton from './controls/ToggleAdvancedSettingsButto
 import shouldShowMissingPasswordWarning from '@joplin/lib/components/shared/config/shouldShowMissingPasswordWarning';
 import MacOSMissingPasswordHelpLink from './controls/MissingPasswordHelpLink';
 const { KeymapConfigScreen } = require('../KeymapConfig/KeymapConfigScreen');
+import FontSearch from './FontSearch';
 
 const settingKeyToControl: any = {
 	'plugins.states': control_PluginsStates,
 };
 
+interface FontData {
```

Comment:
Why "FontData"? Everything is data. It's just a Font

Reply:
I thought it would be better to define the type as it mentioned in the [docs](https://developer.mozilla.org/en-US/docs/Web/API/FontData), but no problem I will change it